nvi-commons/src/main/java/no/sikt/nva/nvi/common/db/CandidateDao.java

```diff
@@ -304,6 +305,26 @@ public String instanceType() {
             }
         }
 
+        @Deprecated
+        //TODO: Remove after migration
```

Comment:
Just add 

```
    @Deprecated
    private static Instant migrationMethodName(Instant instant) {
        if (isNull(instant)) {
            return Instant.now();
        } else {
            return instant;
        }
    }
```
Use the method in the getters and builders. The method can be edited afterwards to return the input and inlined.